code/src/MessageInput.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import './MessageInput.css'
+
+export const MessageInput = () => {
+
+    const MESSAGES_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
+    const [message, setMessage] = useState('')
+
+    const handleSubmit = event => {
+        event.preventDefault()
+
+        fetch(MESSAGES_URL,
+            {
+                method: "POST",
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({ message: message })
```

Comment:
Or we can simplify this (since both left and right side of object have same name) :

```
body: JSON.stringify({ message })
```